src/p2p/net_node.h

```diff
@@ -119,7 +119,7 @@ namespace nodetool
     size_t get_outgoing_connections_count();
     peerlist_manager& get_peerlist_manager(){return m_peerlist;}
     void delete_connections(size_t count);
-    virtual bool block_host(const epee::net_utils::network_address &adress, time_t seconds = P2P_IP_BLOCKTIME);
+    virtual bool block_host(const epee::net_utils::network_address &address, time_t seconds = P2P_IP_BLOCKTIME);
```

Comment:
This might break stuff further down the line.

Reply:
It makes no difference since this is only a function declaration. The argument name is never referenced.